joblib/test/test_numpy_pickle.py

```diff
@@ -378,14 +470,23 @@ def test_joblib_pickle_across_python_versions():
     class SubArray(np.ndarray):
 
         def __reduce__(self):
-            return (_load_sub_array, (np.asarray(self), ))
-
+            return _load_sub_array, (np.asarray(self), )
 
     def _load_sub_array(arr):
         d = SubArray(arr.shape)
         d[:] = arr
         return d
 
+    class ComplexTestObject:
+        """A complex object containing multiple np array memebers."""
+
+        def __init__(self):
+            """Initialize different types of arrays."""
```

Comment:
I'd remove the docstring given that it doesn't bring anything really useful and it is a test object constructor anyway ...